### DISCUSSION TITLE
HTTP 502 error when trying access INL conda channel

### DISCUSSION THREAD

**Discussion by stephen-dixon** [idaholab/moose#28885] [Q&A Getting Started] [answered]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

I can't currently access the INL conda channel to create a new conda-based moose installation. 

The down for everyone website seems to say this isn't just me [here](https://downforeveryoneorjustme.com/conda.software.inl.gov?proto=https)

Is this a known issue and do you know when service might resume? 

### (Optional) code in question / simulation log / errors

```shell
CondaHTTPError: HTTP 502 BAD GATEWAY for url <https://conda.software.inl.gov/public/osx-arm64/current_repodata.json>
Elapsed: 00:00.643889
CF-RAY: 8d4878fdcb0c7d1c-EWR

A remote server error occurred when trying to retrieve this URL.

A 500-type error (e.g. 500, 501, 502, 503, etc.) indicates the server failed to
fulfill a valid request.  The problem may be spurious, and will resolve itself if you
try your request again.  If the problem persists, consider notifying the maintainer
of the remote server.
```

**Comment by TheGreatCid** [comment]
I came here for a sanity check to make sure it wasn't just me. Just have to wait for it to come back up

**Reply by pecan-zq** [reply]
same error for me!

**Reply by TLWise** [reply]
Same here. Sanity checks are good.

**Reply by mmunerato** [reply]
Me too. Same problem starting from Friday.

**Comment by loganharbour** [comment]
Thanks for the report. We're aware of an outage and are investigating it.

**Reply by xbryne** [reply]
Is there any time line currently?

**Comment by milljm** [comment, marked as answer]
https://conda.software.inl.gov/public is back online.

**Comment by aswin-polyenergetics** [comment]
Hi, the server appears unavailable. 

Sharing logs from following the instructions per https://mooseframework.inl.gov/getting_started/installation/moose_conda_binary.html


```
(base) ubuntu@ip-xxx-xx-xx-xx:~$ conda config --add channels https://conda.software.inl.gov/public
(base) ubuntu@ip-xxx-xx-xx-xx:~$ conda create -n moose moose
Channels:
 - https://conda.software.inl.gov/public
 - conda-forge
Platform: linux-64
Collecting package metadata (repodata.json): failed

CondaHTTPError: HTTP 502 Bad Gateway for url <https://conda.software.inl.gov/public/linux-64/repodata.json>
Elapsed: 00:01.084970
CF-RAY: 9cc45e48dc2c2cc2-DFW

A remote server error occurred when trying to retrieve this URL.

A 500-type error (e.g. 500, 501, 502, 503, etc.) indicates the server failed to
fulfill a valid request.  The problem may be spurious, and will resolve itself if you
try your request again.  If the problem persists, consider notifying the maintainer
of the remote server.
```

The website itself doesn't seem available:
<img width="936" height="131" alt="image" src="https://github.com/user-attachments/assets/6eb4b109-83a2-480a-b264-b8ca4c38e27c" />

Please help - has the server has been changed?

**Reply by GiudGiud** [reply]
@loganharbour do you know?

**Reply by loganharbour** [reply]
I'm not sure if my comment here didn't post - but our servers were off for updates. Should be online again.

**Reply by aswin-polyenergetics** [reply]
It is loading fine now - Thank you! @loganharbour @GiudGiud
